ddtrace/contrib/structlog/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""
+Datadog APM traces can be integrated with the logs produced by structlog by:
+
+1. Having ``ddtrace`` patch the ``structlog`` module. This will add a
+processor in the beginning of the chain that adds trace attributes
+to the event_dict
+
+2. Ensure the configuration has at the minimum a processor chain that outputs
+to a log location without errors.
```

Comment:
Is there a scenario where a processor chain raises an exception while generating logs? Is this something we need to call out?